src/main/java/ca/corefacility/bioinformatics/irida/constraints/impl/ProjectMetadataRoleValidator.java

```diff
@@ -3,43 +3,40 @@
 import ca.corefacility.bioinformatics.irida.constraints.ProjectRoleHasCorrectMetadataRole;
 import ca.corefacility.bioinformatics.irida.model.enums.ProjectMetadataRole;
 import ca.corefacility.bioinformatics.irida.model.enums.ProjectRole;
+
 import org.apache.commons.beanutils.BeanUtils;
 
 import javax.validation.ConstraintValidator;
 import javax.validation.ConstraintValidatorContext;
 
 /**
- * Hibernate validator to validate if a user/usegroup is a manager on a project then their metadata role
- * should be set to the highest level. Otherwise if a user/usergroup is a collaborator on a project
- * then their metadata role can be any of the levels (must not be null).
+ * Hibernate validator to validate if a user/usegroup is a manager on a project then their metadata role should be set
+ * to the highest level. Otherwise if a user/usergroup is a collaborator on a project then their metadata role can be
+ * any of the levels (must not be null).
  */
 public class ProjectMetadataRoleValidator implements ConstraintValidator<ProjectRoleHasCorrectMetadataRole, Object> {
-    private String firstFieldName;
-    private String secondFieldName;
+	private String firstFieldName;
```

Comment:
Can you rename this to `projectRoleFieldName` and set the value to `projectRole` by default.

Reply:
Renamed in 3c3136c4c1ab2e318d6a50d5a9639281bbdbb16a